### ISSUE TITLE
Analyser AD0001 Throws Exception VW2019

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/efcore#16285]
After upgrading to VS2019 the analyzer "Microsoft.EntityFrameworkCore.RawSqlStringInjectionDiagnosticAnalyzer" is throwing an exception and showing as a warning in the error list in VS.

_This issue has been moved from https://developercommunity.visualstudio.com/content/problem/518155/analyser-ad0001-throws-exception-vw2019.html
VSTS ticketId: 839596_
_These are the original issue comments:_

Tony Silvestri on 4/7/2019, 00:42 PM (79 days ago): <p>Same error here</p>

Visual Studio Feedback System on 4/8/2019, 00:14 AM (78 days ago): <p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

Steven Hargrave on 5/4/2019, 05:02 PM (51 days ago): <p>Same problem here VS2019</p>

Visual Studio Feedback System on 5/21/2019, 00:00 AM (35 days ago): <p>Thank you for sharing your feedback! Our teams prioritize action on product issues with broad customer impact. See details at:  https://docs.microsoft.com/en-us/visualstudio/ide/report-a-problem?view=vs-2017#faq . In case you need answers to common questions or need assisted support, be sure to use  https://visualstudio.microsoft.com/vs/support/ . We'll keep you posted on any updates to this feedback.</p>

_These are the original issue solutions:_
(no solutions)

**Comment by ajcvickers** [comment]
What version of EF Core are you using?

**Comment by lajones** [comment]
@divega Yes, feel free to transfer to the EFCore repo as needed.

TL;DR At the moment you can only choose 1 github repo to which to transfer the bugs from VS Feedback. I choose EF6 because it was likely to be the one we needed most. I'm investigating whether we can set up to transfer to more than one github repo, but for the moment we might have to put up with transferring twice for some bugs :(

**Comment by ajcvickers** [comment]
**EF Team Triage:** Closing this issue as the requested additional details have not been provided and we have been unable to reproduce it.

*BTW this is a canned response and may have info or details that do not directly apply to this particular issue. While we'd like to spend the time to uniquely address every incoming issue, we get a lot traffic on the EF projects and that is not practical. To ensure we maximize the time we have to work on fixing bugs, implementing new features, etc. we use canned responses for common triage decisions.*
